src/sync/sync_user.hpp

```diff
@@ -230,6 +232,11 @@ friend class SyncSession;
     {
         return m_sync_manager;
     }
+
+    /// Retrieves a general-purpose service client for the Realm Cloud service
+    /// @param service_name The name of the cluster
+    app::MongoClient mongo_client(const std::string& service_name);
```

Comment:
I'm not sure any of the SDKs actually use `SyncUserContext` anymore. @nirinchev does .NET?